composer.json

```diff
@@ -34,13 +34,9 @@
         }
     },
     "scripts": {
-        "post-root-package-install": [
-            "php -r \"file_exists('.env') || copy('.env.example', '.env');\""
-        ],
-        "post-create-project-cmd": [
-            "php artisan key:generate"
-        ],
         "post-install-cmd": [
+            "php -r \"file_exists('.env') || copy('.env.example', '.env');\"",
+            "php artisan key:generate",
```

Comment:
Wouldn't this re-generate the key every time you install your dependencies? I think this might break production in that case and clear all sessions. I think it's best that we leave this as Laravel's default template.